CHANGELOG.md

```diff
@@ -6,19 +6,9 @@
 
 ### DEPENDENCIES
 
-- Add the [IBC Rate Limit module](https://github.com/Stride-Labs/ibc-rate-limiting) 
```

Comment:
isn't this dependency still valid? the module is still required in go.mod as far as I can see.

Reply:
I think adding a module doesn't count as a dependency. It could though. Not that important though given that we have this as features. If we bump the rate limit, then we'll add a changelog entry under dependencies.